msg/sensor_combined.msg

```diff
@@ -23,3 +23,6 @@ int32 baro_timestamp_relative		# timestamp + baro_timestamp_relative = Barometer
 float32 baro_alt_meter			# Altitude, already temp. comp.
 float32 baro_temp_celcius		# Temperature in degrees celsius
 
+# Pre-flight consistency check metrics. These will be zero if the vehicle only has one IMU sensor or is armed.
```

Comment:
Can you add a new topic instead? `sensor_combined` is logged at full rate, and thus every small change will effect log file size & needed bandwidth.